examples/native/ios/Podfile.lock

```diff
@@ -335,4 +335,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 4a7dc44821794ba1f939de367e8bab895789f210
 
-COCOAPODS: 1.7.4
+COCOAPODS: 1.9.3
```

Comment:
the podfile changes don't effect this issue so it would be better to remove these changes